### PR TITLE
Flcc fix

### PR DESCRIPTION
**Pull request by BarracudaPff**
See: [ML-154](https://youtrack.jetbrains.com/issue/ML-154)

### CODE REVIEW

File: tests/unit_tests/test_manual.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_russian():
     model = yttm.BPE.train(TRAIN_DATA_PATH, MODEL_PATH, 50)
     tokenized_text = model.encode([test_text], output_type=yttm.OutputType.SUBWORD)
     expected_result = [
-        ["▁с", "обранный", "▁с", "об", "ран", "и", "е", "▁", "п", "р", "и", "бор"]
+        ["▁", "        ", "с", "об", "ранный", " ", "с", "об", "ран", "и", "е", " ", "п", "р", "и", "бор", "▁", "        "]
```
Comment (BarracudaPff) [inline tests/unit_tests/test_manual.py:23]: Please make sure this result is expected (also in other tests)

File: tests/unit_tests/test_stress.py [hunk 2]
```diff
@@ -25,6 +27,8 @@ def compile_test():
     run(command, check=True, shell=True)
 
 
+# noinspection PyUnresolvedReferences
+@pytest.mark.skip(reason="broken after new implementation")
```
Comment (BarracudaPff) [inline tests/unit_tests/test_stress.py:31]: It looks like `test_stress` and `test_parallel` are broken bcs of `bpe.cpp` fix. Please verify that they can be skipped.
@SokolovYaroslav You can find test output at `Test YouTokenToMe` build config in pinned builds

File: setup.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 def setup_version() -> str:
     version_prefix = "1.0"
     version = os.environ.get("BUILD_NUMBER") if bool(os.environ.get("TEAMCITY_VERSION")) else "SNAPSHOT"
-    return f"{version_prefix}.{version}"
+    return version_prefix + "." + version
```
Comment (SokolovYaroslav) [inline setup.py:11]: ???
Ты старый питон юзаешь?

Comment (SokolovYaroslav) [reply setup.py:11]: Вижу, да

Comment (SokolovYaroslav) [reply setup.py:11]: В других местах тоже есть f-strings. Может лучше в setup.py изменить версию питона лучше на 3.6?

File: tests/unit_tests/test_manual.py (hunk 1 above)
Comment (SokolovYaroslav) [reply tests/unit_tests/test_manual.py:23]: Yes, lgtm

File: tests/unit_tests/test_stress.py (hunk 2 above)
Comment (SokolovYaroslav) [reply tests/unit_tests/test_stress.py:31]: That is expected, yes

**Review by SokolovYaroslav — CHANGES_REQUESTED** [review]
I think we should just change Python3.5 to Python3.6 in setup.py and it can be merged

File: setup.py (hunk 3 above)
Comment (BarracudaPff) [reply setup.py:11]: set default from 3.6

**Review by SokolovYaroslav — APPROVED** [review]

**Pull request merged**